quiche/src/lib.rs

```diff
@@ -7197,6 +7220,18 @@ impl<F: BufFactory> Connection<F> {
                     if ex_data.tx_cap_factor != self.tx_cap_factor {
                         self.tx_cap_factor = ex_data.tx_cap_factor;
                     }
+
+                    if let Some(discover) = ex_data.pmtud {
+                        let path = self.paths.get_active_mut()?;
+
+                        path.pmtud.enable(discover);
+                        // It's harmless to set probe size even if we're disabling
+                        // pmtud.
+                        path.pmtud.set_probe_size(
+                            self.recovery_config.max_send_udp_payload_size,
```

Comment:
probe size has to be larger than max_send_udp_payload_size for pmtud to do anything.

We need a new parameter that can be set per experiment.